guides/get-started/transactions.md

```diff
@@ -112,6 +112,61 @@ try {
 }
 ```
 
+```go
+package main
+
+import (
+	"github.com/stellar/go/build"
+)
+
+func main () {
+	source := "SCZANGBA5YHTNYVVV4C3U252E2B6P6F5T3U6MM63WBSBZATAQI3EBTQ4"
+	destination := "GA2C5RFPE6GCKMY3US5PAB6UZLKIGSPIUKSLRB6Q723BM2OARMDUYEJ5"
+
+	// Make sure destination account exists
+	if _, err := horizon.DefaultTestNetClient.LoadAccount(destination); err != nil {
+		panic(err)
+	}
+
+	passphrase := network.TestNetworkPassphrase
+
+	tx, err := build.Transaction(
+		build.Network{passphrase},
```

Comment:
I think the more idiomatic thing would be to use 'build.TestNetwork' rather than 'build.Network{passphrase}.'